zlib-ng@2.1.6

aaronmondal opened the pull request.

bazel-io commented:
Hello @Vertexwahn, modules you maintain (zlib-ng) have been updated in this PR. Please review the changes.

Wyverald commented on modules/zlib-ng/2.1.6/patches/zlib-ng-bazel-overlay.patch:
```diff
@@ -0,0 +1,1626 @@
+new file mode 100644
+index 00000000..8f382606
+--- /dev/null
++++ b/MODULE.bazel.lock
```
you shouldn't need to patch in the lockfile here (it doesn't do anything anyway). similarly for the github workflow yaml files. Could you maybe add a file filter to whatever script you generated this patch with?

aaronmondal commented:
@Wyverald Thanks for taking a look. I removed the unnecessary files. The previous patch was just created with `git show > xxx.patch`, but for some reason the patch file format was invalid (CI said "can't patch WORKSPACE.bazel").

I removed the redundant files and used a `git diff > xxx.patch` instead.

aaronmondal commented:
@Wyverald Ah I believe the reason the patch failed was that the `WORKSPACE.bzlmod` didn't contain any content, and so the info

```diff
--- /dev/null
+++ b/WORKSPACE.bzlmod
```

wasn't part of the patch. I added an empty line. That should fix it.

fmeum commented:
@aaronmondal Windows is failing.
